src/preprocess.py

```diff
@@ -119,7 +120,7 @@ def remove_stop_words_collection(collection: OrdDict[str, List[str]], stop_word_
     """
     stp = load_stop_words(stop_word_path)
     new_corpus = {}
```

Comment:
this is not an OrderedDict !
![image](https://user-images.githubusercontent.com/1516762/79465934-3d96ca80-7ffc-11ea-8ef7-1a5341a373b2.png)